test/inductor/test_aot_inductor.py

```diff
@@ -2646,7 +2645,6 @@ def fail_non_abi_compatible_cuda(is_skip=False):
     "test_dynamic_cat": fail_minimal_arrayref_interface(),
     # https://github.com/pytorch/pytorch/issues/122978
     "test_dynamic_scalar": fail_stack_allocation(is_skip=True),
-    "test_dynamic_smem_above_default_limit": fail_with_and_without_stack_allocation(),
```

Comment:
These tests don't fail anymore with this diff, so we need to remove them from the list of expected failures..